phpBB/phpbb/di/extension/core.php

```diff
@@ -26,7 +26,11 @@
 */
 class core extends Extension
 {
-	const TWIG_OPTIONS_POSITION = 7;
+	/**
+	 * Index of array in service template.twig.environment inside services_twig.yml
+	 * @var int
+	 */
+	const TWIG_OPTIONS_POSITION = 8;
```

Comment:
I think you also need to update the comment on line 89

Reply:
Yes thanks